### ISSUE TITLE
Arrow keys don't work

### ISSUE THREAD

**Issue by ricardomv** [papyros/terminal-app#20]
Using the arrow keys in the terminal window doesn't work

**Comment by zigapk** [comment]
Any details about your configuration? Because it works perfectly on my computer.

**Comment by ricardomv** [comment]
@zigapk i don't think so i have the latest papyros/qml-material  4af0e33 and papyros/qmltermwidget 031db29 with qt 5.5.1 on arch

**Comment by zigapk** [comment]
@ricardomv I have no idea why. Are you trying to run it in papyros shell?

**Comment by ricardomv** [comment]
no x11 gnome shell

**Comment by ricardomv** [comment]
i just ran a bisect and the issue started when we switched do our fork of qmltermwidget in 7981d8320

**Comment by zigapk** [comment]
Probably, however it works fine on my ubuntu 15.10 (unity on mir). But it is true that I have the original qmltermwidget installed too. There is probably no way this could be the problem, but maybe you could try if that solves the issue?

**Comment by ricardomv** [comment]
yes it works with the upstream version, but that does not solve the issue.

**Comment by zigapk** [comment]
Sure. But I am not sure if I understood your answer correctly. Does the latest commit work with both qmltermwidgets installed on your system?

**Comment by ricardomv** [comment]
i installed qmltermwidget from the arch community repo and changed `/src/qml/TerminalTab.qml` to use it with `import QMLTermWidget 1.0` and it works with the latest commit here. So either we switch to the upstream version or we need to find what is causing this bug.

**Comment by zigapk** [comment]
@iBeliever did the switch to out fork so he will probably know it best?

**Comment by iBelieve** [comment]
@ricardomv have you tried with the upstream master of qmltermwidget, or just the latest release in Arch?

**Comment by ricardomv** [comment]
@iBeliever it works with the latest upstream revision 051e019
